examples/example.js

```diff
@@ -15,7 +15,8 @@ const schema = {
 
 fastify
   .get('/', schema, function (req, reply) {
-    reply(null, { hello: 'world' })
+    reply.header('Content-Type', 'application-json').code(200)
+    reply.reply(null, { hello: 'world' })
```

Comment:
Personally I prefer `reply.send(error, body)`.
I don't like too much the idea of exposing gazillions of apis.

Regarding the promises, *I feel that bro*.